tests/tests/proposal_tests.cpp

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
While it is generally a good idea to put some structure into our testsuite, I think this particular file only adds confusion, because we already have a lot of tests around proposals and most of them are not in this file. I suggest to move this single test elsewhere.

Reply:
The other file that contains a number of proposal tests is authority_tests.cpp. It seemed to me that they had a lot to do with committee stuff, so I didn't put it there. I'll move it there, and avoid this extra file.